### PR TITLE
DOC: further cleanup (followup PR)

### PR DESCRIPTION
**Pull request by klauer**
## Description
Follow-up to #193 

## Motivation and Context
Late - but still appreciated! - review by @ZryletTC :)

## How Has This Been Tested?
Built docs locally.

## Where Has This Been Documented?
It is documentation.

## Questions
Thoughts on the top-level sections? 
https://github.com/pcdshub/happi/pull/193#discussion_r552330581

Built docs from last PR were pushed up manually: https://pcdshub.github.io/happi

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]

**Comment by ZryletTC** [comment]
I published the review after seeing you'd already merged but hadn't fully read through everything yet so I'll do that tonight when I get home and if I find anything else, it can also get included here.

**Comment by ZLLentz** [comment]
I have no real comments/opinions on the top-level section titles

File: docs/source/client.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ information to be stored.
     db = JSONBackend(path='doc_test.json', initialize=True)
 
 If you are connecting to an existing database you can pass the information
-directly into the ``Client`` itself at `__init__``. See :ref:`db_choice`
+directly into the `Client` itself at ``__init__``. See :ref:`db_choice`
 about how to configure your default backend choice
```
Comment (klauer) [inline docs/source/client.rst:35]:
```suggestion
about how to configure your default backend choice.
```

File: docs/source/client.rst [hunk 2]
```diff
@@ -203,3 +203,11 @@ chosen backend directly, but in order to save time you can create an
...
 tell the library to assume you want to use the :class:`.MongoBackend`.
 Otherwise, the library uses the :class:`.JSONBackend`.
+
+..
+   Remove test file created by initializing a JSONBackend above
+
+.. ipython:: python
+   :suppress:
+
+   rm doc_test.json
```
Comment (klauer) [inline docs/source/client.rst:213]:
```suggestion
   rm -f doc_test.json
```

File: happi/backends/json_db.py [hunk 3]
```diff
@@ -240,31 +228,29 @@ def comparison(name, doc):
...
-            _id
-
-        SearchError:
-            If insert is False, but there is no device with the provided _id
-
-        PermissionError:
-            If the write operation fails due to issues with permissions
+        DuplicateError
+            If ``insert`` is `True`, but there is already a device with the
+            provided ``_id``.
```
Comment (klauer) [inline happi/backends/json_db.py:246]: Blank lines between exceptions?

**Comment by klauer** [comment]
Looking good, @ZryletTC - you certainly went the distance with style guide adherence here on what seems like all docstrings. 👏

File: docs/source/containers.rst [hunk 4]
```diff
@@ -52,7 +52,7 @@ Editing the information for a container is a simple as:
 .. note::
 
     :class:`happi.Device` class is **deprecated** due to ambiguous name,
-    conflicting with :class:`ophyd.Device`.
+    conflicting with :class:`ophyd.Device`. HappiItem should be used instead.
```
Comment (ZLLentz) [inline docs/source/containers.rst:55]:
```suggestion
    conflicting with :class:`ophyd.Device`.
    :class:`happi.HappiItem` should be used instead.
```

File: happi/loader.py [hunk 5]
```diff
@@ -88,28 +88,27 @@ class is then instantiated with the given args and kwargs provided.
 
     Parameters
     ----------
-    device : happi.Device
-
-    attach_md: bool, optional
-        Attach the container to the instantiated object as `md`
-
-    use_cache: bool, optional
+    device : .happi.Device
```
Comment (ZLLentz) [inline happi/loader.py:91]:
```suggestion
    device : .happi.HappiItem
```

**Review by ZLLentz — APPROVED** [review]

File: happi/loader.py (hunk 5 above)
Comment (klauer) [reply happi/loader.py:91]: `happi.HappiItem`? (no `.`)

**Comment by klauer** [comment]
Once tests pass, let's get this in. If @ZryletTC wasn't done - there can always be follow-up docs PRs.

**Comment by ZryletTC** [comment]
Yeah, sounds good. I just wanted to commit you suggestions and merge. There's still more I wanted to do but was planning on doing them in a separate PR anyway.

**Review by ZryletTC — APPROVED** [review]

File: happi/backends/json_db.py (hunk 3 above)
Comment (ZryletTC) [reply happi/backends/json_db.py:246]: No, I don't think so. numpydoc isn't explicit about it but none of their examples for other sections have blank lines between.

**Pull request merged**